.github/merge-bot.yml

```diff
@@ -0,0 +1 @@
+enabled: true
```

Comment:
?! I see no staging branch in this repo

Reply:
We don't need a stage for this. The README is fairly detailed.

> Lambda that processes all SES bounces, complaints and deliveries, sends metrics to DataDog and forwards to a filterable SNS queue

SES is Amazon Simple Email Service and this lambda consolidates all bounces/complaints/deliveries into a filterable SNS queue so that applications (and Mark) can see and respond to bounces.

Reply:
@Omicron7 @knutsenm Thanks for clarifying that for me. I don't think it hurts anything to keep the merge-bot.yml file there, though it doesn't do anything. I can remove it if you would rather.